src/modules/Category/index.tsx

```diff
@@ -30,17 +30,17 @@ export const Category = () => {
   const [organizationId, setOrganizationId] = useState<number | ''>(
     userAuth?.organizations?.[0]?.id || ''
   );
-
   const [category, setCategory] = useState<string>('');
   const [departmentId, setDepartmentId] = useState<number>(
     userAuth?.organizations?.[0]?.department_id || 1
   );
   const [priority, setPriority] = useState<number>(0);
+  const [slaUnit, setSlaUnit] = useState<number | ''>('');
```

Comment:
Since the type of this state should be a number, can we have a more appropriate initial value, `null`, `undefined` or `NaN`?